### ISSUE TITLE
GitLab artifacts in GitHub repo

### ISSUE THREAD

**Issue by atapley** [mitrefireline/simharness#2]
There are things mirrored from the GitLab repo that should not be in the GitHub repo, such as `gitlab-ci.yml` and `.gitlab`. Is there a way we can remove these from the GitHub repo without affecting the GitLab repo? Or should we just remove all gitlab aspects and focus on the GitHub side?

**Comment by doyled-it** [comment]
> There are things mirrored from the GitLab repo that should not be in the GitHub repo, such as `gitlab-ci.yml` and `.gitlab`. Is there a way we can remove these from the GitHub repo without affecting the GitLab repo? Or should we just remove all gitlab aspects and focus on the GitHub side?

Yeah, so I was in the middle of converting the GitLab CI jobs to GitHub Workflows, so that we don't have to work on GitLab anymore. But I was getting hung up on a few frustrating things bc workflows are worse.

I was going to remove the GitLab CI stuff at the same time as the workflows getting added in.

**Comment by afennelly-mitre** [comment]
I second the decision to remove the GitLab CI stuff once the GitHub workflows are added (and verified to be working). This is why I did not remove the GitLab artifacts in 216ef259f610546878d9df45f39de3883911c335
